src/ghostwriter/models/substitution.py

```diff
@@ -0,0 +1,32 @@
+"""Substitution parameters for a route transformation."""
+
+from dataclasses import dataclass
+
+from rsp_jupyter_client import RSPJupyterClient
+
+
+@dataclass
+class Parameters:
```

Comment:
This is fine, but because it's a generic name, `RouteTransformationParameters` could be used if you don't mind longer names.

Reply:
I do mind that much longer, and you're always importing `Parameters` from `ghostwriter.model.substitution` so I feel like it's fairly obvious what it's for in context.